src/docfx/docset/Document.cs

```diff
@@ -77,6 +77,28 @@ public string ReadText()
             }
         }
 
+        public override int GetHashCode()
+        {
+            // todo: add docset for calculation
+            return StringComparer.OrdinalIgnoreCase.GetHashCode(FilePath);
```

Comment:
On Linux or mac, path should not be ignore case.

Reply:
We need a discuss about that, will consider it later

---
In reply to: [184596792](https://github.com/dotnet/docfx/pull/2696#discussion_r184596792) [](ancestors = 184596792)